app/src/main/java/com/example/zar/shopistantcontent/MapActivity.java

```diff
@@ -0,0 +1,102 @@
+package com.example.zar.shopistantcontent;
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+public class MapActivity extends AppCompatActivity implements View.OnClickListener{
+    Button a01,a02,a03,a04,a05,a06,a07,a08,a09,a10,a11,a12,a13,a14;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_map);
+        initComponent();
+        a01.setOnClickListener(this);
+        a02.setOnClickListener(this);
+        a03.setOnClickListener(this);
+        a04.setOnClickListener(this);
+        a05.setOnClickListener(this);
+        a06.setOnClickListener(this);
+        a07.setOnClickListener(this);
+        a08.setOnClickListener(this);
+        a09.setOnClickListener(this);
+        a10.setOnClickListener(this);
+        a11.setOnClickListener(this);
+        a12.setOnClickListener(this);
+        a13.setOnClickListener(this);
+        a14.setOnClickListener(this);
+    }
+    public void initComponent(){
```

Comment:
there should be extra empty line in between the end of **onCreate** and **initComponent** functions